src/averageAges.js

```diff
@@ -37,7 +41,14 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren === undefined
```

Comment:
try to rewrite with this condition 👇 
```suggestion
  const women = withChildren
```